std/php/_std/haxe/io/Bytes.hx

```diff
@@ -165,7 +165,7 @@ class Bytes {
 		return b;
 	}
 
-	public inline function toHex() : String {
+	public inline function ofHex() : String {
```

Comment:
`toHex` is the already existing method. You need to add the new `ofHex`, which uses `hex2bin`